python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, inLayer, outDb, outUri, parent=None):
         # updates of UI
         self.setupWorkingMode(self.mode)
         self.cboSchema.currentIndexChanged.connect(self.populateTables)
+        self.widgetSourceSrid.setOptionVisible(QgsProjectionSelectionWidget.CrsNotSet, True)
```

Comment:
I'm wondering - do we need to show the "not set" options? later we just show an error if they are used.

Reply:
I agree but I did this to hide the "invalid projection" when the check-box is unchecked

Reply:
Hm - I just tested this branch, and at the moment this is what I see:

1. open the dialog - crs options are blank and corresponding checkboxes are unticked
2. tick a checkbox - the crs option is still blank

This seems odd to me. If the checkbox is ticked then we should be forcing a choice in the crs selector, not giving an empty crs.

I'd suggest:
- default to project CRS as the initial (disabled) choice
- if a layer from the project is picked from the selector, reset the widgets to match that layer's projection

Reply:
I understand, I changed to show the project CRS by default. For layer's projection, the source code is bit confusing. Can we let like this? It's better than before.